pkg/v1/tarball/write.go

```diff
@@ -302,13 +316,13 @@ func calculateTarballSize(refToImage map[name.Reference]v1.Image, mBytes []byte)
 		if err != nil {
 			return size, fmt.Errorf("unable to get manifest for img %s: %v", name, err)
 		}
-		size += CalculateTarFileSize(manifest.Config.Size)
+		size += calculateSingleFileInTarSize(manifest.Config.Size)
 		for _, l := range manifest.Layers {
-			size += CalculateTarFileSize(l.Size)
+			size += calculateSingleFileInTarSize(l.Size)
 		}
 	}
 	// add the manifest
-	size += CalculateTarFileSize(int64(len(mBytes)))
+	size += calculateSingleFileInTarFileSize(int64(len(mBytes)))
```

Comment:
```suggestion
	size += calculateSingleFileInTarSize(int64(len(mBytes)))
```

Reply:
Need to update the func def as well, right? I can just update the commit and re-push.

Reply:
You have `calculateSingleFileInTarSize` in three places and `calculateSingleFileInTarFileSize` in one place. Whichever you prefer, just needs to be consistent. Also feel free to reject my suggestion and just update/rebase whatever 👍